src/langcheck/metrics/en/pairwise_text_quality.py

```diff
@@ -61,203 +45,64 @@ def pairwise_comparison(
             the score is the same when Model A and Model B are swapped. This is
             useful for ensuring that the evaluator's position bias is not
             impacting the scores. Default True.
-        model_type: The type of model to use ('openai', or 'azure_openai'),
-            default 'openai'
-        openai_client: OpenAI or AzureOpenAI client, default None. If this is
-            None, we will attempt to create a default client.
-        openai_args: Dict of additional args to pass in to the
-            ``client.chat.completions.create`` function, default None
-        use_async: Whether to use the asynchronous API of OpenAI, default False
+        eval_model: The EvalClient instance used for the evaluation. This is
+            marked as Optional because of the backwards compatibility with the
+            previous order of the arguments, but you need to pass this argument
+            to make the function work.
```

Comment:
Hmm, I don't fully agree because it is not just for backwards compatibility but also for consistency with other metrics. For the metrics that offers `'local'` option, I think it is the right design to pass `'local'` as the default option. If we keep this for metrics with 'local', even if users try to use all the metrics with OpenAI, they need to pass the `EvalClient` in a different position of the arguments, which doesn't make much sense to me.

Making this a kwarg also opens the possibility of adding `local` models for some metrics without further breaking the backward compatibility in the future.

Reply:
Ah yeah that's true, if we add a `local` option it'll make sense to keep this `eval_model` arg in its current place. Maybe we can update the docstring here to reflect that? E.g. "This is marked as Optional so that it can follow the above arguments that have default values (for consistency with the other metrics), but this is in fact a required argument."

(Another option would be to make it like this, but it's probably a bit overkill 😅)
```
def pairwise_comparison(
        generated_outputs_a: List[str] | str,
        generated_outputs_b: List[str] | str,
        prompts: List[str] | str,
        sources_a: Optional[List[str] | str] = None,
        sources_b: Optional[List[str] | str] = None,
        reference_outputs: Optional[List[str] | str] = None,
        enforce_consistency: bool = True,
        *,
        eval_model: EvalClient) -> MetricValue[Optional[float]]:
```

Reply:
SGTM, updated the docstrings following your idea!!

(Yeah I think making this keyword-only argument is a bit too much... let me go with the docstring update).